modules/policy-as-code-about.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * operating/manage_security_policies/custom-security-policies.adoc
+:_mod-docs-content-type: CONCEPT
+[id="policy-as-code-about_{context}"]
+= Managing polices as code
+
+[role="_abstract"]
+
+You can create and manage policies as code by saving policies as Kubernetes custom resources (CRs) and applying them to clusters by using a Kubernetes-native continuous delivery (CD) tool such as Argo CD.
+
+:FeatureName: Policy as code
+include::snippets/technology-preview.adoc[]
+
+{product-title-short} provides the ability to use default policies or create custom policies for your system. With the policy as code feature, you can create custom policies by authoring them locally and then using a continuous delivery tool such as Argo CD to track, manage, and apply policies to your clusters that are running {product-title-short}. You can also use the API to configure connections to your own GitOps repository such as github. To author policies locally, you create CRs that represent the desired state of the policies. After you create or update CRs and use the CI/CD tool to apply them, the policies stored in the {product-title-short} database are created or updated.
+
+Policy as code is useful for Kubernetes security architects who want to author policies in YAML or JSON instead of using the {product-title-short} portal. It is also useful to GitOps administrators who already manage Kubernetes configurations by using a GitOps workflow.
+
+{product-title-short} installs a new configuration controller in the namespace where Central is installed, typically the `stackrox` namespace. With an Argo CD workflow, you configure Argo CD to communicate with this controller in the `stackrox` namespace by using the Kubernetes API. After you configure this connection, the controller in {product-title-short} receives information from the Kubernetes API about new, updated, or deleted policies that are managed as individual Kubernetes CR files. {product-title-short} reconciles the policy CR to the policy stored in the {product-title-short} database.
+
+//architecture diagram
+
+With a GitOps workflow that does not use Argo CD, you configure your GitOps repository to connect to Central in {product-title-short} through the {product-title-short} API. A CR is not used.
+//not sure I understand why no CR, isn't a policy saved as a CR even when using GitOps?
```

Comment:
Sometimes leaving no space between // and the next character might cause the content to be added in instead of ignored.